src/2025h1/all-hands.md

```diff
@@ -0,0 +1,95 @@
+| Book the venue                                 | [RustNL]                       | Done! |
+| Catering / snacks / food / drinks              | [RustNL]                       | Done! |
+| [Register] for the Rust All-Hands 2025         | *every project member*         | Majority already signed up! |
+| Send out conformations/tickets                 | Mara                           |       |
+| Send out detailed information                  | Mara                           |       |
+| Answer logistical questions                    | Mara                           |       |
+| Interface between project and RustNL           | Mara                           |       |
+| Make hotel reservations                        | [RustNL]                       | In progress |
+| Book hotel                                     | *all participants*             | RustNL will provide suggestions |
+| Book travel                                    | *all participants*             |       |
+| Team swag                                      |                                |       |
+| ↳ Decide on what to get (stickers? shirts?)    | ![Team][] [leadership-council] | Suggestions very welcome! |
+| ↳ Acquire it                                   | [RustNL]                       |       |
+| Make plans for what to do at the all-hands     | *every team*                   |       |
+| ↳ Gather input from your teams                 | *team leads* (or delegate)     |       |
+| ↳ Provide input for planning team              | *team leads* (or delegate)     |       |
+| ↳ Make an agenda for your team's room          | *team leads* (or delegate)     |       |
+| ↳ Coordinate the overall process               | Planning team                  | Small group of 2-3 people. Volunteers welcome! |
+| ↳ Make a room plan (after gathering input)     | Planning team                  |       |
+| Prepare one or two plenary sessions            | ![Team][] [leadership-council] |       |
+| Submit talks for the "Rust Project Track"      | *project members*              | Possibility to give talks at the conference before the all-hands. |
+| Organise an optional "pre all-hands day"       | Mara, [RustNL]                 | An optional day without an agenda, with space for social activities |
+| ↳ Acquire gifts (secret!)                      | Mara                           | Done! |
+| Moderation / safety                            | [RustNL] and moderator team    |       |
+| Accessibility and covid safety                 | [RustNL]                       |       |
+| Come to the Rust All-Hands 2025                | *all participants*             |       |
+| Reimburse travel costs                         | Rust Foundation                |       |
+
+[RustNL]: https://rustweek.org/about/
+[Register]: https://docs.google.com/forms/d/e/1FAIpQLSeeoEsZGLGokSK0Gn8GVRHhM23KgbwKsp67oxi79KfdsbipkQ/viewform
+
+## Frequently asked questions
+
+### I'm so excited about the all hands!
+
+Me too!
```

Comment:
```suggestion
Me too!

### How will we schedule meetings?

@jackh726 asked:

>  The key challenge here is going to be scheduling meetings for teams with overlapping membership. Likely it'll probably make sense to stagger team meetings such that there are blocks of time with very few overlaps of "official" meetings (with the acknowledgement that maybe that means during those times the parallelism goes down).

@m-ou-se answered:

> Yeah that's part of the reason for a "Planning team" to "Coordinate the overall process". Hopefully we can make the draft agendas public early so teams can work together directly to align their plans a bit. It's going to be a fun puzzle though. ^^

### Who can attend?

@jackh726 asked:

> One thing I haven't seen listed is who other than the Project is/may attend? From what I remember, there is a separate space available - it would be really good to have an actual list for that, which is available to Project members, since people may be interested in chatting/coordinating.

@m-ou-se answered:

> There will be an unconference at the same venue that will host groups like the embedded working group, Rust for Linux, Bevy maintainers, UI/App developers, and a few other groups. That part is handled by RustNL as a mostly separate event, that just happens to take place in the same location.
>
> RustNL will of course share which groups that will be once they confirm, so we can coordinate potential collaboration with these groups. But from a organisation (and funding) perspective, we treat the all-hands and the unconference as mostly separate events, which is why I didn't include it in the project goal.

```